verilog/formatting/verilog_token.cc

```diff
@@ -563,6 +563,7 @@ FTT GetFormatTokenType(yytokentype e) {
       case '+':
       case '*':
       case '-':
+      case '&':
```

Comment:
You can tell this group of cases is missing other operators like `%`, `^`, `/`, `|`, ...
If you like, feel free to mark these as TODO in comments, open new issue, so they can be handled later.
If you feel up for fixing these and adding test cases in this PR, I'd accept that as well.

Reply:
I've added the operators you listed as well as some bitwise binary operators (`|` and `^`).

In addition to that I started adding more tests and noticed that `~|` and `~&` were defined as binary, but these operators are unary in all context so I added a patch for that too.